.env.example

```diff
@@ -49,15 +49,15 @@ FILE_AGENT_LLM="openai"
 SUGGESTIONS_LLM="openai"
 
 # model
-ANSWER_AGENT_MODEL="gpt-4o mini"
-INTENT_AGENT_MODEL="gpt-4o mini"
+ANSWER_AGENT_MODEL="gpt-4o-mini"
+INTENT_AGENT_MODEL="gpt-4o-mini"
 VALIDATOR_AGENT_MODEL="mistral-large-latest"
-DATASTORE_AGENT_MODEL="gpt-4o mini"
-MATHS_AGENT_MODEL="gpt-4o mini"
-WEB_AGENT_MODEL="gpt-4o mini"
-CHART_GENERATOR_MODEL="gpt-4o mini"
-ROUTER_MODEL="gpt-4o mini"
-FILE_AGENT_MODEL="gpt-4o mini"
-SUGGESTIONS_MODEL="gpt-4o mini"
-REDIS_HOST="redis"
+DATASTORE_AGENT_MODEL="gpt-4o-mini"
+MATHS_AGENT_MODEL="gpt-4o-mini"
+WEB_AGENT_MODEL="gpt-4o-mini"
+CHART_GENERATOR_MODEL="gpt-4o-mini"
+ROUTER_MODEL="gpt-4o-mini"
+FILE_AGENT_MODEL="gpt-4o-mini"
+SUGGESTIONS_MODEL="gpt-4o-mini"
+REDIS_HOST="localhost"
```

Comment:
Can we move REDIS condig somewhere else in this file? I don't think it fits under `# model`

Reply:
It's done, I have moved it up.